shared/whatYouNeed/raspberrypi.md

```diff
@@ -3,7 +3,7 @@
 * A 4GB or larger SD card. All the Raspberry Pi, except older model Bs, uses a microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
```

Comment:
I think "all models of Raspberry Pi" might sound better than "All the Raspberry Pi" ? And "uses" should be "use".